README.md

```diff
@@ -88,12 +88,12 @@ headers.
 where a user must manually initiate profiling, AutoProfile™ automatically schedules and continuously performs profiling appropriate for
 critical production environments.
 
-Please refer to the [Instana Go Collector docs](https://www.instana.com/docs/ecosystem/go/#instana-autoprofile) to learn how to activate and
+Please refer to the [Instana Go Collector docs](https://www.ibm.com/docs/en/obi/current?topic=go-collector-common-operations#instana-autoprofile%E2%84%A2) to learn how to activate and
```

Comment:
For some reason, this link is not opened properly.
The page opens correctly, but the anchor doesn't seem to be working as intended.